indra/newview/llagentbenefits.cpp

```diff
@@ -195,6 +195,25 @@ S32 LLAgentBenefits::getTextureUploadCost(const LLImageBase* tex) const
     return getTextureUploadCost();
 }
 
+// <FS:Chanayane> 2048x2048 snapshots upload to inventory
```

Comment:
We don't use this type of tags in the official Second Life viewer code. Your authorship on the changes will be preserved via git anyway.

Otherwise, the code looks good. Please remove the tags and I'll gladly accept it.